src/BugDisplay.php

```diff
@@ -42,8 +46,79 @@ public function displayBug() {
 				return;
 			}
 
+			$attributes = $data->attributes;
+			$data->attributes = [];
+			foreach ($attributes as $attr => $value) {
+				$data->attributes[] = $value + ["field" => $attr];
+			}
+
 			$data->id = (int) $routerInfo["id"];
 			$res->end($this->mustache->render("bug.mustache", new TemplateContext($req, $data)));
 		};
 	}
+
+	public function submitBug() {
+		return function (Request $req, Response $res, $routerInfo) {
+			$data = yield from $this->manager->submit($req, $routerInfo);
+			if ($data["success"] ?? false) {
+				$res->setStatus(302);
+				$res->addHeader("Location", "/{$data['id']}");
```

Comment:
I think all bug related URIs should be in root. The main point of a bug tracker is tracking bugs, pretty redundant to prefix /bugs.